deep_tempering/__init__.py

```diff
@@ -1,4 +1,4 @@
 from deep_tempering.training import EnsembleModel
-from deep_tempering.training_test import model_builder as example_model_builder
+# from deep_tempering.training_test import model_builder as example_model_builder
```

Comment:
why is this?

Reply:
why would we want to include objects from `training_test` in `deep_tempering` module?